tests/postreview.test.js

```diff
@@ -25,7 +25,7 @@ describe(validateReview, () => {
     const review = {
       "data": {
         "comment": "test",
-        "rating": 5,
+        "rating": 0,

```

Comment:
If I'm reading this correctly, this change doesn't really change anything with regards to whether the test will succeed or fail.